src/mainFilesSidebarLoader.js

```diff
@@ -52,7 +51,8 @@ const isEnabled = function(fileInfo) {
 // server, so it is done here too just to be safe.
 let tabInstance = null
 
-window.addEventListener('DOMContentLoaded', () => {
+window.addEventListener('DOMContentLoaded', async () => {
```

Comment:
Oh 🙈,  since Vue is imported dynamically in the mainFilesSidebar.js file, guestNameStore ends up trying to access it before it is initialized, causing the error. We can go with dynamic imports inside the components ( unless there are circular imports )